clients/venice-thin-client/src/main/java/com/linkedin/venice/client/schema/RouterBasedStoreSchemaFetcher.java

```diff
@@ -33,7 +33,7 @@
 public class RouterBasedStoreSchemaFetcher implements StoreSchemaFetcher {
   public static final String TYPE_KEY_SCHEMA = "key_schema";
   public static final String TYPE_VALUE_SCHEMA = "value_schema";
-  public static final String TYPE_UPDATE_SCHEMA = "update_schema";
+  public static final String TYPE_GET_UPDATE_SCHEMA = "update_schema";
```

Comment:
So we can't change the router endpoint to use a less confusing name?

Reply:
Has this router endpoint already been used?

Reply:
The endpoint is already being used by our Samza beam users

Reply:
It's not obvious here but the HTTP method can also help disambiguate this